infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/KubernetesServerExposer.java

```diff
@@ -186,12 +186,14 @@ public void expose(Map<String, ? extends ServerConfig> servers) throws Infrastru
 
     provisionServicesForDiscoverableServers(servers);
 
-    Service service = createService(internalServers, externalServers, unsecuredPorts, securedPorts);
+    Optional<Service> serviceOpt = createService(internalServers, unsecuredPorts);
```

Comment:
you can use `createService(internalServers, unsecuredPorts).ifPresent(service -> doYourStuffHere())`

Reply:
yeah, lambdas are fine until you didn't have exceptions inside... :+1:

Reply:
good point. I always think of nice compact lambda, until I hit our checked exceptions everywhere :/ bummer